src/communication/pipe.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PIPE_HPP_
+#define PIPE_HPP_
+
+#include <aos/common/tools/string.hpp>
+
+#include "transport.hpp"
+
+namespace aos::zephyr::communication {
+
+class Transport : public TransportItf {
+public:
+    static constexpr auto cXSReadPath  = CONFIG_AOS_CHAN_TX_PATH;
```

Comment:
CONFIG_AOS_CHAN_TX_PATH is not defined. In general we can use same config for vchan and pipes. But it should be in Kconfig.

Reply:
why not defined, i renamed previouse `CONFIG_AOS_VCHAN_TX_PATH` to more general

Reply:
Done